spec/system/submissions/show_spec.rb

```diff
@@ -44,7 +44,7 @@
     end
 
     scenario 'coach visits submission show', js: true do
-      sign_in_user coach.user, referer: timeline_event_path(submission_pending)
+      sign_in_user coach.user, referer: review_timeline_event_path(submission_pending)
```

Comment:
With the re-organization in this  PR, this file is:

1. In the wrong place.
2. Named incorrectly.
3. Contains incorrect scenario titles and
4. Does not contain specs for the new submission show page that should actually be in this particular file.